src/Sulu/Bundle/DomainEventBundle/Entity/DomainEvent.php

```diff
@@ -0,0 +1,234 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\DomainEventBundle\Entity;
+
+use Sulu\Component\Security\Authentication\UserInterface;
+
+class DomainEvent
```

Comment:
should we add an `additionalData` property to the class that is persisted and allows to add some project-specific data via an event-listener inside of a project?

Reply:
also it might be worth noting that it is not possible to overwrite this class inside of a project with the current implementation. not sure if that is something that should be possible. if yes, we would need to make the `DomainEvent` class configurable to implement some kind of factory that intanciates the configured class